src/covid_model_seiir_pipeline/forecasting/task.py

```diff
@@ -1,5 +1,5 @@
 from argparse import ArgumentParser, Namespace
```

Comment:
all changes in file pulled in from https://github.com/ihmeuw/covid-model-seiir-pipeline/pull/68